sjb/config/test_cases/test_pull_request_image_registry_e2e.yml

```diff
@@ -1,5 +1,5 @@
 ---
```

Comment:
Can you also rename this file to `test_pull_request_registry_extended.yml`?

Reply:
Sorry, `test_pull_request_image_registry_extended.yml`

Reply:
@kargakis sure. done.